From 6b41a8a337db852ec47635fcd724d73c9e1046c1 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Fri, 25 Nov 2016 11:34:18 +0100
Subject: [PATCH] Add vim modelines to source files

This automatically sets up vim to use two space indenting if modelines
is active.

Signed-off-by: Phil Sutter <psutter@redhat.com>
---
 src/libkeepalive.c | 1 +
 test/test.c        | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/libkeepalive.c b/src/libkeepalive.c
index a08bd98abe7b8..06b66ca7a8ea5 100644
--- a/src/libkeepalive.c
+++ b/src/libkeepalive.c
@@ -1,3 +1,4 @@
+/* vim: set ts=2 sw=2 et: */
 /*
    _ _ _     _                         _ _
   | (_) |__ | | _____  ___ _ __   __ _| (_)_   _____
diff --git a/test/test.c b/test/test.c
index 756d7ae631f4a..c793225eaa820 100644
--- a/test/test.c
+++ b/test/test.c
@@ -1,3 +1,4 @@
+/* vim: set ts=2 sw=2 et: */
 /*
    _ _ _     _                         _ _
   | (_) |__ | | _____  ___ _ __   __ _| (_)_   _____
-- 
2.10.0

